akka-docs-dev/rst/scala/code/docs/stream/ReactiveStreamsDocSpec.scala

```diff
@@ -92,7 +92,7 @@ class ReactiveStreamsDocSpec extends AkkaSpec {
 
     //#source-publisher
     val authorPublisher: Publisher[Author] =
-      Source(tweets).via(authors).runWith(Sink.publisher)
+      Source(tweets).via(authors).runWith(Sink.publisher(1))
```

Comment:
I don't want to do that for 2 reasons:

1) that would mean we'd have to use overloading for Java
2) that would also mean that given the path of least resistance, people would use the parameterless one without thinking and then would wonder why they can only attach a single subscriber.